chain-signatures/node/src/protocol/signature.rs

```diff
@@ -587,7 +589,8 @@ impl SignatureManager {
             return;
         }
         let mut failed_presigs = Vec::new();
-        while let Some(mut presignature) = {
+        let mut alternate = false;
```

Comment:
why is alternate back?

Reply:
alternate is back because of how presignatures are taken now. This one actually is more concise since we actually do alternate truly this time, but it was alternate before too just different conditional logic